engines/order_management/app/services/order_management/stock/estimator.rb

```diff
@@ -23,7 +24,7 @@ def shipping_rates(package, frontend_only = true)
         shipping_rates.sort_by! { |r| r.cost || 0 }
 
         unless shipping_rates.empty?
-          if frontend_only
+          if checkout
             shipping_rates.each do |rate|
               if rate.shipping_method.frontend?
```

Comment:
I found that shipping_rates scope frontend is dead code and so is ShippingMethod.on_frontend_query.
We could delete these two things.


We could also rename shipping_method.frontend? to shipping_method.checkout? so we finish the refactor I started of renaming frontend to checkout.

I think the constance DISPLAY is shipping_method.rb is dead code:
DISPLAY = [:both, :front_end, :back_end].freeze

I checked the DBs and we dont have front_end shipping_methods, the display_on value is either empty, nil or back_end.
This means the scope  shipping_method.on_backend_query is wrong and should test for empty string instead of 'front_end', like this (like the scope display_on_checkout):
"#{table_name}.display_on != '' OR #{table_name}.display_on IS NULL"

Reply:
This comment above is all parallel points to the changes in this PR. Not exactly part of the PR :+1:

Reply:
Yes!! I noticed this as well. There is definitely a lot of cleanup that we could do around `on_frontend_query`, `front_end`, `frontend?` etc etc etc.

Reply:
I'm going to mark this as resolved and create https://github.com/openfoodfoundation/openfoodnetwork/issues/6636